Environment-Management/SDDC_Documentation.md

```diff
@@ -0,0 +1,1032 @@
+ 
+> **NOTE**: During the test, the cluster is capable of handling node failures and site failures because the storage is unaffected, and all volumes are in the ONLINE state. 
+
+To revert back to normal operation, follow a similar procedure which involves stopping the PROD_DR database and starting the PROD database.
+
+1. ConfD jobs to stop the database in DC2 and start the database in DC1:
+    ```
+    confd_client db_stop db_name: PROD_DR 
+    confd_client db_start db_name: PROD
+    ```
+2. Activate the backup schedule(s) as described in [Activating Backup Schedules](#activating-or-deactivating-backup-schedules) for the PROD database
+3. Deactivate backup schedule(s) for the PROD_DR database (see [Activating Backup Schedules](#activating-or-deactivating-backup-schedules)). 
+
+The next L-1 will run using the previously created L-0 backup from the weekend. After the L-1 backup is done, consider deleting the L-0 backup from the passive database (DC2) to free up space in advance.
+
+---
+### Disaster of active data center (DC1)
+
+Cluster overview before the data center failure:
+
+![Standard SDDC setup](images/SDDC/SDDC_standard.png)
+
+
+During a Disaster Scenario, all nodes on the active site in DC1 become unavailable. As a result, the active database in DC1 crashes due to multiple active node failures. All data which was last successfully committed in the active database is also written redundantly on the passive side. The only data loss would therefore be data which was not yet committed or in the process of being committed when the database crashed.
+
+Cluster overview after DC1 (active site) failed:
+
+![Overview after DC1 fails](images/SDDC/dr_active_1.png)
+
+> **CAUTION:** The cluster requires manual intervention from here. While a disaster-case is normally that one Data Center is unavailable (or at least two active nodes have failed), it actually may also be the case that the link between the two data centers is severed, so each site of the cluster thinks the other has failed (split-brain). For this reason, a process, team or runbook decides which set of nodes to mark as suspended or which steps to execute next.
+
+> **NOTE:** To operate an Exasol cluster, the cluster needs a quorum. If the cluster has no quorum, the cluster does not accept any changes to any of the services and for example also confd will not show the right data or does not respond to requests properly. To create a quorum at least 50% + 1 of the nodes of a cluster need to be online. 
+
+In this example, all nodes from DC1 failed and this will make the quorum of the cluster fail. In order to restore the quorum with the remaining nodes from DC2 the failed nodes need to be suspended (temporarily removed from the cluster quorum).
+
+> **NOTE:** It might take up to 60 seconds for the cluster to reevaluate the quorum and to recognize failed nodes. Reelections are constantly ongoing every couple of seconds.
+
+1. Use ConfD to suspend the failed nodes from DC1:
```

Comment:
We should add a note here that this is a critical operation.


Specifically, this is safe to execute if you're suspending at most 50% of nodes, and you only do it in one DC (as is the case here).

If you do it on both sides, or if the other side has 50% + 1 node for whatever reason, you _will_ get data corruption.